### PR TITLE
feat: show possible values for option in help output

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
feature

**Did you add tests for your changes?**
Yes updated snaps
**If relevant, did you update the documentation?**
No
**Summary**
show possible values for option in help output

Screenshot 2021-07-05 at 5 20 47 PM

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

File: packages/webpack-cli/lib/webpack-cli.js [hunk 1]
```diff
@@ -1363,10 +1363,21 @@ class WebpackCLI {
                     );
                 }
 
+                const flag = this.getBuiltInOptions().find(
+                    (flag) => option.long === `--${flag.name}`,
+                );
+
+                if (flag && flag.configs[0].values) {
```
Comment (alexander-akait) [inline packages/webpack-cli/lib/webpack-cli.js:1370]: I think we can have more them, i.e. `config[1]`/`config[2]`, I think better use `map` here and collect them

**Review by alexander-akait — COMMENTED** [review]
Good idea, one note

File: packages/webpack-cli/lib/webpack-cli.js (hunk 1 above)
Comment (snitin315) [reply packages/webpack-cli/lib/webpack-cli.js:1370]: Will update 👍

**Review by alexander-akait — APPROVED** [review]

**Comment by alexander-akait** [comment]
Feel free to merge

**Comment by snitin315** [comment]
Need one more review. /cc @webpack/cli-team

**Review by evenstensberg — APPROVED** [review]

**Pull request merged**